src/main/scala/com/campudus/tableaux/database/model/TableauxModel.scala

```diff
@@ -170,14 +156,22 @@ class TableauxModel(override protected[this] val connection: DatabaseConnection)
     }
   }
 
-  private def insertNormalValue[A, B <: ColumnType[A]](tableId: TableId, columnId: ColumnId, rowId: RowId, value: A): Future[Cell[A, B]] = for {
+  private def insertMultilanguageValues[A <: JsonObject](tableId: TableId, columnId: ColumnId, rowId: RowId, value: A): Future[Cell[Any]] = {
```

Comment:
`insertMultiLanguageValues` would be better (you are using `MultiLanguage` camel case elsewhere)